pimd/pim6_cmd.c

```diff
@@ -392,6 +392,30 @@ DEFPY (interface_no_ipv6_mroute,
 					    source_str);
 }
 
+DEFPY (interface_ipv6_pim_passive,
+       interface_ipv6_pim_passive_cmd,
+       "[no$no] ipv6 pim passive",
+       NO_STR
+       IPV6_STR
+       PIM_STR
+       "Disable exchange of protocol packets\n")
+{
+	VTY_DECLVAR_CONTEXT(interface, ifp);
+	struct pim_interface *pim_ifp = ifp->info;
+
+	if (!pim_ifp) {
+		vty_out(vty, "PIM not enabled on this interface\n");
+		return CMD_SUCCESS;
+	}
```

Comment:
We should just go a head an enable pim on this interface in passive mode.

Reply:
Done